Sushi.Mediakiwi.Data.Elastic/UI/NotificationList.cs

```diff
@@ -0,0 +1,135 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using Sushi.Mediakiwi.AppCentre.Data.Implementation;
+using Sushi.Mediakiwi.Framework;
+using Sushi.Mediakiwi.UI;
+using System;
+using System.Collections.Generic;
+using System.Dynamic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Sushi.Mediakiwi.Data.Elastic.UI
+{
+    public class NotificationList : BaseImplementation
+    {
+        private Repositories.NotificationRepository _repository;
+
+        public NotificationList(Repositories.NotificationRepository repository)
+        {
+            _repository = repository;
+
+            ListSearch += NotificationList_ListSearch;
+            ListLoad += NotificationList_ListLoad;            
+        }
+
+        /// <summary>
+        /// Gets or sets the date.
+        /// </summary>
+        /// <value>The date.</value>
+        [Framework.ContentListItem.TextLine("Date")]
+        public string Date { get; set; }
+
+        /// <summary>
+        /// Gets or sets the type.
+        /// </summary>
+        /// <value>The type.</value>
+        [Framework.ContentListItem.TextLine("Type")]
+        public string Type { get; set; }
+
+        /// <summary>
+        /// Gets or sets the note.
+        /// </summary>
+        /// <value>The note.</value>
+        [Framework.ContentListItem.TextLine("Notification")]
+        public string Note { get; set; }
+
+        private async Task NotificationList_ListLoad(ComponentListEventArgs arg)
+        {
+            // todo: get index from url
```

Comment:
Looks like this is done, please remove comment

Reply:
yes

Reply:
just checked, it is not done, because those parameters are not supplied by MK. in fact, the whole list is not working, it is just a first prototype which can be used to build a real working list. 

this due to MK's incapability of working with composed keys ( {string, string} ), or even string based ID's at all.